scripts/build-circt.sh

```diff
@@ -24,12 +29,12 @@ while [[ "$#" -ge 1 ]] ; do
 	case "$1" in
 		--build-path)
 			shift
-			CIRCT_BUILD="$1"
+			CIRCT_BUILD="${PWD}/$1"
```

Comment:
Should the default not be `${CIRCT_BUILD_DIR}`

Reply:
This is user input and not the default.